components/gearScript/zen/fn_zen_createLoadoutLocker.sqf

```diff
@@ -0,0 +1,45 @@
+#include "../macros.hpp"
+
+CLIENT_ONLY;
+
+params ["_position", "_unit"];
+
+if ((!isNull _unit) and {_unit isKindOf "CAManBase"}) exitWith
+{
+    ["Cannot use this module on units.  Try on ground, vehicles or containers."] call zen_common_fnc_showMessage;
+};
```

Comment:
What happens if the function is called on, say, an object (e.g. by placing the Zeus module on a vehicle)? This condition won't catch that, so the function continues to run, causing the dialog to pop Not critical, but... odd? Is this intended?